config/routes.rb

```diff
@@ -48,6 +48,7 @@
   resources :featured_circuits, only: %i[index create]
   delete "/featured_circuits", to: "featured_circuits#destroy"
 
+  get "/users/edit", to: redirect('/')
```

Comment:
why ?

Reply:
Devise provides this route and it is currently enabling users to change their emails and passwords (without the forgot mail system), so disabling access to this.